src/Distribution/src/Manager.php

```diff
@@ -14,11 +14,9 @@ final class Manager implements MutableDistributionInterface
      * @var array<string, UriResolverInterface>
      */
     private array $resolvers = [];
-    private string $default;
 
-    public function __construct(string $name = self::DEFAULT_RESOLVER)
+    public function __construct(private string $default = self::DEFAULT_RESOLVER)
```

Comment:
We can't rename parameters in public signatures in v3

Reply:
sure, reverted, updated the configured rule as disallow rename property https://github.com/spiral/framework/pull/1139/commits/ccd058aae09a77189f31f1b45b20203664efb925